modules/solid_mechanics/src/userobjects/CrackFrontDefinition.C

```diff
@@ -22,6 +22,7 @@ void addCrackFrontDefinitionParams(InputParameters& params)
 {
   MooseEnum direction_method("CrackDirectionVector CrackMouth CurvedCrackFront");
   MooseEnum end_direction_method("NoSpecialTreatment CrackDirectionVector", "NoSpecialTreatment");
+  params.addParam<std::vector<RealVectorValue> >("crack_front_points","Set of points to define crack front");
```

Comment:
Reading in a param that's a vector of RealVectorValues way makes it really easy for me to parse a set of points that are defined as triplets of values.  For instance, the user can pass in three points like this:
```crack_front_points = '0 -10 .5
                        0 -10 0
                        0 -10 -.5'```
and I can parse it with one line:
```_crack_front_points = getParam<std::vector<RealVectorValue> >("crack_front_points");```
Are you suggesting that I should be doing something different here? I'm happy to change it if needed.